-meeting-template.md

```diff
@@ -1,22 +1,42 @@
-Toronto Workers Co-op Meeting 
-=========================================
+---
+filename: -meeting-template.md
+---
+# YYYY-MM-DD Toronto Workers Co-op: All-Hands Meeting
 
-[🗓 Calendar](https://calendar.google.com/calendar/embed?src=s2224p8sptnujs736vplf9anjo%40group.calendar.google.com&ctz=America%2FToronto)  
-[📑 Retreat Planning Board](https://github.com/cryptographydog/december-meetup/projects/1)  
+<sup>[from template][template]</sup>
 
-Time:   
-Location: https://appear.in/offline  
-Attending:  
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
 
-[TOC]
+Time: ...
```

Comment:
Can we add two trailing spaces so the line break is rendered in MD?

Reply:
Same for the two lines below!

Reply:
oh! i had to idea about that! I've been removing them like a doofus!